Examples/Algorithms/TrackFinding/src/TrackFindingAlgorithm.cpp

```diff
@@ -698,4 +694,54 @@ ProcessCode TrackFindingAlgorithm::finalize() {
   return ProcessCode::SUCCESS;
 }
 
+// TODO this is somewhat duplicated in AmbiguityResolutionAlgorithm.cpp
+// TODO we should make a common implementation in the core at some point
+void TrackFindingAlgorithm::computeSharedHits(
+    TrackContainer& tracks, const MeasurementContainer& measurements) const {
+  // Compute shared hits from all the reconstructed tracks
+  // Compute nSharedhits and Update ckf results
+  // hit index -> list of multi traj indexes [traj, meas]
+
+  std::vector<std::size_t> firstTrackOnTheHit(
+      measurements.size(), std::numeric_limits<std::size_t>::max());
```

Comment:
Does this only use the total number of measurements and nothing else?

Reply:
I believe so. Before that it was the size of the source link container which is the same. The source link index will then just point into this array.